### PR TITLE
Change from DOCKER_REGISTRY to IMAGE_REGISTRY in some scripts

### PR DESCRIPTION
**Pull request by YangKeao**
### What problem does this PR solve?

In the previous PR of build scripts, I modified the `DOCKER_REGSITRY` environment variable to `IMAGE_REGISTRY` (to keep consistent with other variable like `IMAGE_PROJECT`, `IMAGE_TAG`...). In this PR, I will modify other scripts to fit this.

I will also modify other script inside the jenkins.

### CODE REVIEW

File: hack/e2e.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ Usage: hack/e2e.sh [-h] -- [extra test args]
     -h      show this message and exit
 Environments:
     PROVIDER                    Kubernetes provider, e.g. kind, gke, eks, defaults: kind
-    DOCKER_REGISTRY             image docker registry
+    IMAGE_REGISTRY             image docker registry
```
Comment (STRRL) [inline hack/e2e.sh:37]:
```suggestion
    IMAGE_REGISTRY              image docker registry
```

File: hack/local-up-chaos-mesh.sh [hunk 2]
```diff
@@ -39,7 +39,7 @@ Environments:
     KUBECONFIG            path to the kubeconfig file, defaults: ~/.kube/config
     KUBECONTEXT           context in kubeconfig file, defaults to current context
     NAMESPACE             Kubernetes namespace in which we run our chaos-mesh.
-    DOCKER_REGISTRY       image docker registry
+    IMAGE_REGISTRY       image docker registry
```
Comment (STRRL) [inline hack/local-up-chaos-mesh.sh:42]:
```suggestion
    IMAGE_REGISTRY        image docker registry
```

**Review by STRRL — COMMENTED** [review]
rest LGTM

**Review by STRRL — APPROVED** [review]

**Review by cwen0 — APPROVED** [review]

**Review by Andrewmatilde — CHANGES_REQUESTED** [review]
I think you should also replace `DOCKER_REGISTRY` with `IMAGE_REGISTRY` in doc. https://chaos-mesh.org/docs/production-installation-using-helm/

**Comment by STRRL** [comment]
> I think you should also replace `DOCKER_REGISTRY` with `IMAGE_REGISTRY` in doc. [chaos-mesh.org/docs/production-installation-using-helm](https://chaos-mesh.org/docs/production-installation-using-helm/)

It seems this PR would not affect the helm chart.

**Comment by YangKeao** [comment]
> > I think you should also replace `DOCKER_REGISTRY` with `IMAGE_REGISTRY` in doc. [chaos-mesh.org/docs/production-installation-using-helm](https://chaos-mesh.org/docs/production-installation-using-helm/)
> 
> It seems this PR would not affect the helm chart.

Yes. I should modify the development document https://chaos-mesh.org/docs/configure-development-environment/

**Pull request merged**